pkg/sensors/tracing/generickprobe.go

```diff
@@ -454,6 +469,45 @@ func createGenericKprobeSensor(name string, kprobes []v1alpha1.KProbeSpec) (*sen
 		logger.GetLogger().Infof("Added generic kprobe sensor: %s -> %s", load.Name, load.Attach)
 	}
 
+	if len(multiIDs) != 0 {
```

Comment:
nit: can we move code below into a new function?

Reply:
ok, will do